cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/SlangTestRunner.java

```diff
@@ -107,16 +107,20 @@
         }
         for (Map.Entry<String, SlangTestCase> testCaseEntry : testCases.entrySet()) {
             SlangTestCase testCase = testCaseEntry.getValue();
-            if(testCase == null){
+            if (testCase == null) {
                 failedTestCases.put(testCase, "Test case cannot be null");
                 continue;
             }
-            log.info("Running test: " + testCaseEntry.getKey() + " - " + testCase.getDescription());
-            try {
-                CompilationArtifact compiledTestFlow = getCompiledTestFlow(compiledFlows, testCase);
-                runTest(testCase, compiledTestFlow, projectPath);
-            } catch (RuntimeException e){
-                failedTestCases.put(testCase, e.getMessage());
+            if (CollectionUtils.isEmpty(testCase.getTestSuites()) || CollectionUtils.containsAny(testSuites, testCase.getTestSuites())) {
```

Comment:
No, I check if the test case contains any test suites. For a test case that does not contain any testSuites property, I treat it as base test suite, and run it.
If the testSuites property is not empty, I look for an intersection with the test suites passed in runtime